models/base_model.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+"""
+Module contains the BaseModel class
+"""
+import uuid
+import datetime
```

Comment:
It can be better if we import directly datetime from datetime

So we can use dtetime.now() instead of detetime.datetime.now()